.github/workflows/docker.yml

```diff
@@ -27,17 +31,20 @@ jobs:
       - name: set up qemu
         uses: docker/setup-qemu-action@v1
       - name: set up buildx
+        id: docker_buildx
         uses: docker/setup-buildx-action@v1
       - name: log in to ghcr
         uses: docker/login-action@v1
         with:
-          registry: ghcr.io
-          username: lus
-          password: ${{ secrets.CR_TOKEN }}
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
       - name: build and push
         uses: docker/build-push-action@v2
         with:
+          builder: ${{ steps.docker_buildx.outputs.name }}
+          platforms: linux/amd64, linux/arm64
           push: true
-          tags: ghcr.io/lus/pasty:${{ env.TAG }}
+          tags: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.TAG }}, ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.COMMIT_HASH }}
```

Comment:
This would assemble to `ghcr.io/pasty` in this case (it is missing the author's name) as far as I am aware

Reply:
the `IMAGE_NAME: ${{ github.repository }}`sets `IMAGE_NAME` based on the `github.repository` variable, which includes the author's handle, i.e. `lus/pasty`. For example (https://github.com/williampiv/pasty/runs/4215404430?check_suite_focus=true#step:10:300). Thanks!

Reply:
Ah, I see. I wasn't aware of that. Thank you!